bdasl.src.html

```diff
@@ -37,5 +37,101 @@ <h2>Parsing BDASL CIDs</h2>
         may also be equal to <code>0x1e</code> (BLAKE3) ([[blake3]]).
       </p>
     </section>
+    <section>
+      <h2>Streaming Verification</h2>
+      <p>
+        Section 6.4 of the [[blake3]] spec outlines
+        <i>Streaming Verification</i>, a process that allows the sender
+        and receiver of a CID to incrementally verify data as it is
+        being transferred. Applying this technique to CIDs is the core
+        benefit of BDASL, which is well-suited to both fetching byte
+        ranges within a CID, and verifying data where the cost of
+        faulty transmission will impact the performance of an application.
+        Verified streaming incurs minimal overhead on payloads of all
+        sizes, and scales linearly as a small percentage of the size of
+        the CID being verified.
+      </p>
+      <p>
+        Streaming verification rounds to the nearest kilobyte for
+        verification. For more details and a reference implementation, see
+        [[iroh-blobs]].
+      </p>
+    </section>
+    <section>
+      <h2>Verifying HTTP Range Requests</h2>
+      <p>
+        [[rfc9110]] defines HTTP range requests for fetching a single
+        contiguous set of bytes from a larger source held by a server.
+        Range requests use a Range header: <code>Range: bytes={start}-{end}</code>
```

Comment:
i assume {start} and {end} are just regular old ints that map to byte#s from 0?  prolly doesn't need saying but i tend to err on the side of obnoxiously explicit.

Reply:
yes exactly, I can call that out more explicitly